distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -294,3 +294,9 @@ hazelcast.version.check.enabled=false
 artemis.amqp.port=5671
 artemis.multiprotocol.port=5672
 artemis.openwire.port=61616
+artemis.live.host=0.0.0.0
+artemis.backup.host=0.0.0.0
+#add a list of non local comma separated IP's here that will always be available even if other cluster members are down
+artemis.network.iplist=
```

Comment:
i have tried that but it ended up breaking builds by causing random broker shutdowns